.github/workflows/tests-owners.yml

```diff
@@ -34,16 +34,30 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # empty = release that corresponds to current branch name
-        release: ['', 'eln']
+        # Matrix must be built purely from includes due to bugs (as of 2021-02).
+        # Don't forget to copypaste also the same matrix below in rpm tests!
+        # For various branches, comment out what does not apply:
+        #   on -devel, run everything
+        #   on -release, only -release
+        #   on master, only master and ELN
```

Comment:
This is incorrect.

The correct one should be:

```
on -devel, run master, ELN and -devel
on -release, run everything
on master, only master and ELN
```
That also explains why I had there information of on what branch it should be started. If I read it correctly this will now work only for the one branch + `ELN`.

This should be fixed but let's that solve in separate PR.

Reply:
Changes flow like this:
```
devel -> release
      -> master -> ELN
```
A pull request to release makes the code go nowhere else, so why would I test that on master? On the other hand, anything on devel will get everywhere, so testing it everywhere makes sense.

This PR is for devel, so the full set lands here, and we can just remove things on the other branches after merging.

Reply:
My fault here. I totally forgot that this is `pull_reques_target` trigger.